components/form/autocompleted/README.md

```diff
@@ -25,6 +25,7 @@ The component exposes the following props:
 * handleBlur (Function): This function is called everytime user exits the input.
 * handleFocus (Function): This function is called everytime user focus on the input.
 * selectFirstByDefault (Boolean): **Optional** It sets first position for the autocomplete default active option. Defaults to `true`.
+* closeOnOutsideClick (Boolean): **Optional** Enables close suggestions on click outside feature
```

Comment:
It would be great to document the elements that are not taken into account in order to close the suggestions list.